package.json

```diff
@@ -10,10 +10,10 @@
   ],
   "scripts": {
     "test": "npm run tests-unit",
-    "tests-unit": "mocha $(find . -name '*.spec.ts' ! -name '*.net.spec.*')",
+    "tests-unit": "mocha $(find . -name '*.spec.ts' ! -name '*.local.net.spec.*' ! -name '*.devnet.spec.*' ! -name '*.testnet.spec.*')",
     "tests-localnet": "mocha $(find . -name '*.local.net.spec.ts')",
-    "tests-devnet": "mocha $(find . -name '*.dev.net.spec.ts')",
-    "tests-testnet": "mocha $(find . -name '*.test.net.spec.ts')",
+    "tests-devnet": "mocha $(find . -name '*.devnet.spec.ts')",
+    "tests-testnet": "mocha $(find . -name '*.testnet.spec.ts')",
```

Comment:
Somehow rogue / unrelated change: fixed prefix for system tests. For `localnet` we didn't yet, in order to avoid merge conflicts here:


https://github.com/multiversx/mx-sdk-js-core/pull/389